server/repository/src/db_diesel/log_row.rs

```diff
@@ -0,0 +1,83 @@
+use super::{log_row::log::dsl as log_dsl, StorageConnection};
+
+use crate::{db_diesel::invoice_row::invoice, repository_error::RepositoryError, user_account};
+
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use diesel_derive_enum::DbEnum;
+
+table! {
+    log (id) {
+        id -> Text,
+        log_type -> crate::db_diesel::log_row::LogTypeMapping,
+        user_id -> Nullable<Text>,
+        record_id -> Nullable<Text>,
+        created_datetime -> Timestamp,
```

Comment:
think this should be just datetime. If it is a creation time comes from the log type.